### PR TITLE
Add Ukranian Localization

### PR DESCRIPTION
**Pull request by blessedcoolant**
Also update Italian for 2.3.0.

### CODE REVIEW

File: invokeai/frontend/dist/locales/common/ua.json [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  "discordLabel": "Discord",
+  "settingsLabel": "Налаштування",
+  "darkTheme": "Темна",
+  "lightTheme": "Світла",
+  "greenTheme": "Зелена",
+  "langEnglish": "Англійська",
+  "langRussian": "Російська",
+  "langItalian": "Iталійська",
+  "langPortuguese": "Португальська",
+  "langFrench": "Французська",
```
Comment (ebr) [inline invokeai/frontend/dist/locales/common/ua.json:16]:
```suggestion
  "langFrench": "Французька",
```

File: invokeai/frontend/dist/locales/hotkeys/ua.json [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+  "redoStroke": {
+    "title": "Повторити пензель",
+    "desc": "Повторити мазок пензля"
+  },
+  "resetView": {
+    "title": "Автоматичный вигляд",
+    "desc": "Скинути вигляд полотна"
+  },
+  "previousStagingImage": {
+    "title": "Previous Staging Image",
```
Comment (ebr) [inline invokeai/frontend/dist/locales/hotkeys/ua.json:54]:
```suggestion
    "title": "Попереднє зображення",
```

File: invokeai/frontend/dist/locales/hotkeys/ua.json [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+    "desc": "Попереднє зображення"
+  },
+  "nextStagingImage": {
+    "title": "Next Staging Image",
```
Comment (ebr) [inline invokeai/frontend/dist/locales/hotkeys/ua.json:58]:
```suggestion
    "title": "Наступне зображення",
```

File: invokeai/frontend/public/locales/common/ua.json [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+  "discordLabel": "Discord",
+  "settingsLabel": "Налаштування",
+  "darkTheme": "Темна",
+  "lightTheme": "Світла",
+  "greenTheme": "Зелена",
+  "langEnglish": "Англійська",
+  "langRussian": "Російська",
+  "langItalian": "Iталійська",
+  "langPortuguese": "Португальська",
+  "langFrench": "Французська",
```
Comment (ebr) [inline invokeai/frontend/public/locales/common/ua.json:16]:
```suggestion
  "langFrench": "Французька",
```

File: invokeai/frontend/public/locales/hotkeys/ua.json [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+  "redoStroke": {
+    "title": "Повторити пензель",
+    "desc": "Повторити мазок пензля"
+  },
+  "resetView": {
+    "title": "Автоматичный вигляд",
+    "desc": "Скинути вигляд полотна"
+  },
+  "previousStagingImage": {
+    "title": "Previous Staging Image",
```
Comment (ebr) [inline invokeai/frontend/public/locales/hotkeys/ua.json:54]:
```suggestion
    "title": "Попереднє зображення",
```

File: invokeai/frontend/public/locales/hotkeys/ua.json [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+    "desc": "Попереднє зображення"
+  },
+  "nextStagingImage": {
+    "title": "Next Staging Image",
```
Comment (ebr) [inline invokeai/frontend/public/locales/hotkeys/ua.json:58]:
```suggestion
    "title": "Наступне зображення",
```

**Review by ebr — CHANGES_REQUESTED** [review]
Minor UA fixes suggested

File: invokeai/frontend/dist/locales/common/ua.json (hunk 1 above)
Comment (blessedcoolant) [reply invokeai/frontend/dist/locales/common/ua.json:16]: @netsvetaev Please go through the requested changes and let me know.

**Review by netsvetaev — APPROVED** [review]

**Comment by netsvetaev** [comment]
please add «Давыдова Екатерина» as an author's name for the UA translation.

**Review by netsvetaev — APPROVED** [review]

**Comment by blessedcoolant** [comment]
@lstein Any idea why this is not auto merging?

**Review by lstein — APPROVED** [review]
I'm happy to approve, but I don't speak Ukrainian!

**Review by ebr — APPROVED** [review]

**Comment by blessedcoolant** [comment]
@lstein Can you see if you can merge this? It keeps telling me that there are conflicts but I see none.

**Pull request merged**